cli/admin/src/test/java/io/pravega/cli/admin/controller/SecureControllerCommandsTest.java

```diff
@@ -157,5 +157,13 @@ public void testDescribeStreamCommand() {
         Assert.assertTrue(commandResult.contains("testStream"));
     }
 
+
+    @Test
+    @SneakyThrows
+    public void testDeleteReaderGroupCommand() {
+        String commandResult = TestUtils.executeCommand("controller delete-readergroup _system testRG", cliConfig());
```

Comment:
Is there a test to demonstrate that the CLI command actually works?

Reply:
Updated the test case. Now this testcase verify the deletion of readergroup also.